src/components/LoanCards/GridLoanCard.vue

```diff
@@ -6,15 +6,19 @@
 			:amount="loan.loanAmount"
 			:use="loan.use"
 			:country="loan.geocode.country.name" />
+
+		<MatchingText :matching-text="loan.matchingText" />
```

Comment:
Generally we've been favoring using kebab-case syntax when adding components to the template, as that is more in line with the html spec. In this case: `<matching-text :matching-text="loan.matchingText" />`

Reply:
@emuvente changes made to both the borrower-info and matching-text components.